content/ar/docs/concepts/overview/_index.md

```diff
@@ -0,0 +1,96 @@
+
+يوفر لك كوبيرنيتيس:
+
+* **اكتشاف الخدمات وموازنة الأحمال**
+يمكن لكوبيرنيتيس كشف حاوية تحت اسم نظام أسماء النطاق (DNS) أو تحت عنوان آيبي (IP) خاص بها. إذا كانت حركة المرور إلى التطبيق عالية، فيمكن لكوبيرنيتيس موازنة الأحمال وتوزيع حركة مرور بين عدة حاويات متماثلة بحيث يكون التطبيق مستقراً.
+* **تنسيق التخزين**
+يسمح لك كوبيرنيتيس بتركيب نظام تخزين من اختيارك تلقائياً، مثل المستودعات المحلية وموفري السحابة العامة والمزيد.
+* **عمليات الطرح والتراجع الآلي**
+يمكنك وصف الحالة المرغوبة للحاويات التي تم نشرها باستخدام كوبيرنيتيس ، ويمكنه تغيير الحالة الفعلية إلى الحالة المطلوبة بوتيرة قابلة للضبط. على سبيل المثال، يمكنك إعداد كوبيرنيتيس لإنشاء حاويات جديدة للنشر وإزالة الحاويات الحالية واعتماد جميع مواردها في الحاوية الجديدة بشكل آلي
+* **التعبئة الذكية للعقد**
+زود كوبيرنيتيس بمجموعة من العقد التي يمكنه استخدامها لتشغيل الحاويات. حدد لكوبيرنيتيس متطلبات الحاويات من وحدة المعالجة المركزية والذاكرة (رام). يقوم كوبيرنيتيس بوضع الحاويات في العقد لتحقيق أقصى استفادة من مواردها.
+* **التعافي الذاتي**
+بقوم كوبيرنيتيس بإعادة تشغيل الحاويات الفاشلة، واستبدال الحاويات، ويزيل الحاويات التي لا تستجيب لفحص الصحة الذي يحدده المستخدم، ولا يعلن عنها للعملاء حتى تصبح جاهزة للخدمة.
+* **إدارة ملفات الضبط و السرية**
+يتيح لك كوبيرنيتيس تخزين المعلومات الحساسة وإدارتها، مثل كلمات المرور ورموز أوأوث (OAuth) المميزة ومفاتيح أس أس أيتش (SSH). يمكنك نشر وتحديث الأسرار وتهيئة التطبيق دون إعادة بناء صور الحاوية، ودون الكشف عن الأسرار في مصادر رموز البرامج
+
+## كوبيرنيتيس ليس 
+
+كوبيرنيتيس ليس نظامًا تقليديًا كاملا للمنصة كخدمة. نظرًا لأن كوبيرنيتيس يعمل على مستوى الحاويات بدلاً من مستوى الأجهزة، فإنه يوفر بعض الميزات القابلة للتطبيق بشكل عام والمشتركة في عروض المنصة كخدمة، مثل النشر والقياس وموازنة الحمل ، ويتيح للمستخدمين دمج حلول التسجيل والمراقبة والتنبيه. ومع ذلك، فإن كوبيرنيتيس ليس وحدة متراصة، وهذه الحلول الإفتراضية اختيارية وقابلة للتوصيل. يوفر كوبيرنيتيس اللبنات الأساسية لإنشاء المنصات للمطورين، ولكنه يحافظ على اختيارات المستخدم والمرونة عند الحاجة.
+
+كوبيرنيتيس:
+
+* لا يحد من أنواع التطبيقات المدعومة. يهدف كوبيرنيتيس إلى دعم مجموعة متنوعة للغاية من أحمال العمل، بما في ذلك أحمال العمل عديمة الحالة وذات الحالة وأحمال معالجة البيانات. إذا كان يمكن تشغيل تطبيقاً في حاوية، فأنه سيعمل بشكل رائع على كوبيرنيتيس
+* لا ينشر كود المصدر ولا يبني التطبيق الخاص بك. يتم تحديد مهام سير عمل التكامل المستمر والتسليم والنشر من خلال ثقافات المؤسسة وتفضيلاتها وكذلك المتطلبات الفنية.
+* لا يقدم خدمات على مستوى التطبيق كخدمات مدمجة، مثل البرامج الوسيطة (على سبيل المثال، حافلات الرسائل)، أو إطارات معالجة البيانات (على سبيل المثال سبارك Spark)، أو قواعد البيانات (على سبيل المثال مايسايكل MySQL)، أو ذاكرات التخزين المؤقت، أو أنظمة التخزين العنقودية (على سبيل المثال ساف Ceph). يمكن تشغيل تلك المكونات على كوبيرنيتيس، ويمكن الوصول إليها عن طريق التطبيقات التي تعمل على كوبيرنيتيس من خلال آليات محمولة، مثل [خدمة السمسار المفتوح](https://openservicebrokerapi.org/).
```

Comment:
for later: discuss the translation for "message bus"

Reply:
ok Can you open a discussion thread on cncf glossary